Formula/aom.rb

```diff
@@ -0,0 +1,44 @@
+class Aom < Formula
+  desc "Codec library for encoding and decoding AV1 video streams"
+  homepage "https://aomedia.googlesource.com/aom"
+  url "https://aomedia.googlesource.com/aom", :using => :git, :tag => "v1.0.0", :revision => "d14c5bb4f336ef1842046089849dee4a301fbbf0"
```

Comment:
Use `https://aomedia.googlesource.com/aom.git` as url, which will remove the need for `:using => :git`. Look at other formulas to see how to indent the `:tag` and `:revision` in separate lines

Reply:
Done.